chpl/chpl-api/src/main/java/gov/healthit/chpl/web/controller/SearchController.java

```diff
@@ -166,144 +164,6 @@ public SearchController(ListingSearchService searchService) {
         return searchService.findListings(searchRequest);
     }
 
-    @Deprecated
-    @DeprecatedApi(friendlyUrl = "/search/beta",
```

Comment:
This is annoying, but since we don't have /search/beta anymore, is the plan to rename /search/v2 to just be /search (meaning deprecate the existing endpoint and add a new one called as just "/search"? Or should it be /listings/search or  /certified-products/search? Maybe a bigger question than this PR, I just thought if we want to do that we could deprecate the "v2" one and add the new one at this time).